flake.nix

```diff
@@ -6,24 +6,19 @@
     "An experimental language for high performance safe convenient metaprogramming";
 
   inputs = {
-    by-name.inputs.nixpkgs-lib.follows = "nixpkgs";
-    by-name.url = "github:bb010g/by-name.nix";
     nixpkgs.url = "github:nixos/nixpkgs/nixos-24.11";
     pre-commit-hooks.inputs.nixpkgs.follows = "nixpkgs";
     pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
     rust-overlay.inputs.nixpkgs.follows = "nixpkgs";
     rust-overlay.url = "github:oxalica/rust-overlay";
+    luvit-nix.url = "github:lunnova/luvit-nix/lunnova/update-inputs-and-rpath-patch";
```

Comment:
Ideally the luvit-nix PR is merged first so this can be pointed at its main branch instead of a PR branch

Reply:
Merged and updated to point to @aiverson's repo